package.json

```
@@ -2,18 +2,22 @@
   "name": "solc",
   "version": "0.8.10",
   "description": "Solidity compiler",
-  "main": "index.js",
+  "main": "dist/index.js",
   "bin": {
-    "solcjs": "solcjs"
+    "solcjs": "dist/solc.js"
   },
   "scripts": {
-    "lint": "node ./node_modules/semistandard/bin/cmd.js",
-    "updateBinary": "node downloadCurrentVersion.js && node verifyVersion.js",
-    "prepublishOnly": "npm run updateBinary",
-    "pretest": "npm run lint",
-    "test": "tape ./test/index.js",
-    "coverage": "node ./node_modules/nyc/bin/nyc.js --reporter=lcov --reporter=text-summary ./node_modules/tape/bin/tape ./test/index.js",
-    "coveralls": "npm run coverage && node ./node_modules/coveralls/bin/coveralls.js <coverage/lcov.info"
+    "build": "tsc",
+    "postbuild": "node build/postbuild.js",
+    "lint": "semistandard",
+    "lint-fix": "semistandard --fix",
+    "updateBinary": "ts-node ./downloadCurrentVersion.ts && ts-node ./verifyVersion.ts",
+    "prepublishOnly": "npm run build && npm run updateBinary",
+    "pretest": "npm run lint && npm run build && npm run copy-test-files",
+    "copy-test-files": "cp -r ./test/resources ./dist/test/",
```

I think the hyphenated convention is nicer but we're already using camel case for some commands so I'd stick to it unless there's a good reason not to.
```suggestion
    "copyTestFiles": "cp -r ./test/resources ./dist/test/",
```

---

Updated to use camel case